updates/6/updates.sql

```diff
@@ -233,4 +233,76 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF962B098Ecc43
+UPDATE assets SET cryptocompare="ROSN" where identifier="ROSN";
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("ROSN", "S", "Roseon Finance", "ROSN", 1620551645, NULL, "roseon-finance", "ROSN", "ROSN"); INSERT INTO common_asset_details(asset_id, forked) VALUES("ROSN", NULL);
+UPDATE assets SET coingecko="chronologic" where identifier="_ceth_0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F", "C", "Chronologic", "DAY", 1505774047, NULL, "chronologic", "", "0xE814aeE960a85208C3dB542C53E7D4a6C8D5f60F");
+UPDATE assets SET coingecko="sx-network" where identifier="_ceth_0x99fE3B1391503A1bC1788051347A1324bff41452";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x99fE3B1391503A1bC1788051347A1324bff41452", 18, ""); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x99fE3B1391503A1bC1788051347A1324bff41452", "C", "SportX", "SX", 1610747210, NULL, "sx-network", "", "0x99fE3B1391503A1bC1788051347A1324bff41452");
+UPDATE assets SET coingecko="uniqueone-photo" where identifier="_ceth_0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F";
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F", 18, NULL); INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F", "C", "FOTO", "FOTO", 1612087883, NULL, "unique-photo", "FOTO", "0xF6eA0E60cee427B90aA36f327eE61d1d4CABC49F");
+UPDATE assets SET coingecko="aircoin-2" where identifier="AIR-2";
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("AIR-2", "B", "Air coin", "AIR", 1395273600, NULL, "aircoin-2", "AIR*", "AIR-2"); INSERT INTO common_asset_details(asset_id, forked) VALUES("AIR-2", NULL);
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("SIGNA", "B", "Signum", "SIGNA", 1402531200, NULL, "signum", "SIGNA", "SIGNA"); INSERT INTO common_asset_details(asset_id, forked) VALUES("SIGNA", NULL);
+*
+UPDATE assets SET swapped_for="SIGNA" WHERE identifier="BURST";/*Introduce BURST token swap*/
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("BURST", "B", "Burst", "BURST", 1402531200, "SIGNA", "signum", "SIGNA", "BURST"); INSERT INTO common_asset_details(asset_id, forked) VALUES("BURST", NULL);
```

Comment:
here coingecko/cryptocompare change to signum and SIGNA? Why? If they need to then you also need to include it in the upgrade, otherwise it won't happen in the DB.

Reply:
True. Here I had doubts but coingecko removed the old asset and the migration was one to one, so I believed it made sense to update this fields

Reply:
Okay but what I mean is if you update it in the insertion clause, you also **have to** update it in the update.